### PR TITLE
Add `max_failover_replication_time_lag` configuration with constraints

### PR DESCRIPTION
**Pull request by vsharma6855**
- Set description for `max_failover_replication_time_lag`
- Define value constraints (min: 10, max: 9223372036854776000)
- Provide default (60) and example (50) values

### CODE REVIEW

**Review by loosla — APPROVED** [review]
It looks good to me :+1: 
It would be great if this new field - `max_failover_replication_time_lag` - could be added to godo, doctl, and Terraform.

It seems that `synchronous_replication` and `stat_monitor_enable` are also absent from godo, doctl, and Terraform.

File: specification/resources/databases/models/advanced_config/postgres_advanced_config.yml [hunk 1]
```diff
@@ -407,3 +407,10 @@ properties:
       Enable the pg_stat_monitor extension. <b>Enabling this extension will cause the cluster to be restarted.</b> When this extension is enabled, pg_stat_statements results for utility commands are unreliable.
     type: boolean
     example: false
+  max_failover_replication_time_lag:
+    description: >-
+      Number of seconds of master unavailability before triggering database failover to standby.</b>  The default value is 60.
```
Comment (andrewsomething) [inline specification/resources/databases/models/advanced_config/postgres_advanced_config.yml:412]:
```suggestion
      Number of seconds of master unavailability before triggering database failover to standby. The default value is 60.
```

**Comment by vsharma6855** [comment]
> It looks good to me 👍 It would be great if this new field - `max_failover_replication_time_lag` - could be added to godo, doctl, and Terraform.
> 
> It seems that `synchronous_replication` and `stat_monitor_enable` are also absent from godo, doctl, and Terraform.

Yes, we will be adding PR for that.

**Review by loosla — APPROVED** [review]
:+1:

**Comment by vsharma6855** [comment]
Here is the card for above recommendations:
https://do-internal.atlassian.net/browse/DBAAS-5519

**Review by andrewsomething — APPROVED** [review]

**Pull request merged**